libraries/AP_HAL_ChibiOS/HAL_ChibiOS_Class.cpp

```diff
@@ -279,8 +279,8 @@ void HAL_ChibiOS::run(int argc, char * const argv[], Callbacks* callbacks) const
      *   RTOS is active.
      */
 
-#ifdef HAL_USB_PRODUCT_ID
-  setup_usb_strings();
+#if HAL_USE_SERIAL_USB == TRUE
+    usb_initiailse();
```

Comment:
typo

Reply:
you want the american spelling of initialise?

Reply:
ahh, now I see it. Amazing how blind I can be to spelling errors. Fixed

Reply:
:) I'll be more specific next time